src/App.tsx

```diff
@@ -15,49 +19,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabetically = 'alphabetically',
+  ByLength = 'by length',
+  NoSorting = '',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: SortType,
+  isReversed: boolean,
+): Good[] {
+  const preparedGoods = goods.map((good, index) => ({
+    name: good,
+    id: index + 1,
+  }));
+
+  if (sortField === SortType.Alphabetically
+    || sortField === SortType.ByLength
+  ) {
```

Comment:
```suggestion
  if (sortField) {
```
Just check if `sortField` exists, an empty string is a falsy value: https://developer.mozilla.org/en-US/docs/Glossary/Falsy